cpp/benchmarks/io/parquet/parquet_writer_benchmark.cpp

```diff
@@ -85,7 +86,7 @@ void BM_parq_write_varying_options(benchmark::State& state)
       cudf_io::parquet_writer_options::builder(source_sink.make_sink_info(), view)
         .compression(compression)
         .stats_level(enable_stats)
-        .column_chunks_file_path(file_path);
+        .column_chunks_file_paths(file_paths);
```

Comment:
How about this? No need for additional variable.
```suggestion
        .column_chunks_file_paths({file_path});
```
?